.github/CONTRIBUTION.md

```diff
@@ -11,11 +11,20 @@
 
 Run unit tests with:
 
-`yarn test:unit`
+    yarn test:unit
 
 Run rendering tests with:
 
-`yarn test:rendering`
+    # Install dependencies
+    yarn --frozen-lockfile
+
+    # Build nebula.js visualization
+    yarn build
+
+    chmod 777 ./test/rendering/commands/run-rendering-test.sh
+    sudo ./test/rendering/commands/run-rendering-test.sh
```

Comment:
why do we need sudo?

Reply:
right, no need anymore as we have chmod 777